CHANGES.md

```diff
@@ -1,5 +1,18 @@
+0.14 (unreleased)
+-----------------
+
+- Add `--mpl-results-always` flag which disables removing of test images for
+  tests which pass. Test images are also stored and generated for hash tests
+  when a baseline dir is also provided. [#108]
+
+- Add a `--mpl-generate-summary=json` option which saves a JSON summary of the
```

Comment:
Not sure if this will need a separate entry?
```suggestion
  when a baseline dir is also provided. [#108]

- When generating a HTML summary page, the `--mpl-results-always` flag is
  automatically applied. [#131]

- Add a `--mpl-generate-summary=json` option which saves a JSON summary of the
```